modules/andes-core/broker/src/main/java/org/wso2/andes/kernel/dtx/DtxBranch.java

```diff
@@ -54,7 +54,7 @@
     /**
      * Session Id of a session that is recovered from storage
      */
-    public static final int RECOVERY_SESSION_ID = -1;
+    public static final UUID RECOVERY_SESSION_ID = UUID.randomUUID();
```

Comment:
Actually, we don't need this constant. It is used as the session id for recovered branches.